manifests/charts/istio-control/istio-discovery/templates/validatingwebhookconfiguration.yaml

```diff
@@ -2,11 +2,7 @@
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-{{- if eq .Release.Namespace "istio-system"}}
-  name: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
-{{- else }}
-  name: istio{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}-{{ .Release.Namespace }}
-{{- end }}
+  name: istio{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}-{{ .Values.global.istioNamespace }}
```

Comment:
Can we make this consistent with sidecar-injector?

Reply:
As in make the resource name closer to `istio-sidecar-injector-...`?

Reply:
Yup

Reply:
Changed resource name to `istio-validator-...`, thanks